lib/modules/dosomething/dosomething_helpers/dosomething_helpers.variable.inc

```diff
@@ -227,7 +263,13 @@ function dosomething_helpers_get_variable_names() {
     'count_flagged',
     'count_pending',
     'count_promoted',
+    'external_signup_goal',
+    'external_new_member_goal',
+    'external_impact_goal',
     'goal',
```

Comment:
I know you didn't write this line (I probably did) but this can be removed since this was hot module goals